src/appmixer/hubspot/engagements/CreateEmail/component.json

```diff
@@ -0,0 +1,195 @@
+                            }
+                        ],
+                        "index": 4
+                    },
+                    "hsEmailHtml": {
+                        "type": "text",
+                        "label": "Email HMTL",
+                        "tooltip": "The body of an email if it is sent from a CRM record.",
+                        "index": 5
+                    },
+                    "hsEmailStatus": {
+                        "type": "select",
+                        "label": "Email Status",
+                        "tooltip": "The send status of the email.",
+                        "options": [
+                            {
+                                "label": "Bounced",
+                                "value": "BOUNCED"
+                            },
+                            {
+                                "label": "Failed",
+                                "value": "FAILED"
+                            },
+                            {
+                                "label": "Scheduled",
+                                "value": "SCHEDULED"
+                            },
+                            {
+                                "label": "Sending",
+                                "value": "SENDING"
+                            },
+                            {
+                                "label": "Sent",
+                                "value": "SENT"
+                            }
+                        ],
+                        "index": 6
+                    },
+                    "hsEmailSubject": {
+                        "type": "text",
+                        "label": "Email Subject",
+                        "tooltip": "The subject line of the logged email.",
+                        "index": 7
+                    },
+                    "hsEmailText": {
+                        "type": "text",
+                        "label": "Email Text",
+                        "tooltip": "The subject line of the logged email.",
+                        "index": 8
+                    },
+                    "hsAttachmentIds": {
+                        "type": "text",
+                        "label": "Email Attachment IDs",
+                        "tooltip": "The IDs of the email's attachments. Multiple attachment IDs are separated by a semi-colon.",
+                        "index": 9
+                    },
+                    "hsEmailHeaders": {
+                        "type": "text",
+                        "label": "Email Headers",
+                        "tooltip": "The email's headers. The value for this property will automatically populate certain read only email properties. Learn how to set email headers.",
+                        "index": 10
+                    }
+                }
+            }
+        }
+    ],
+    "outPorts": [
+        {
+            "name": "out",
+            "options": [
+                { "label": "id", "value": "id" },
```

Comment:
Labels are expected in Start Case, that's why the pipeline is failing too. It makes sense for `ID` but I didn't find proper labels for eg `hs_body_preview_is_truncated`.